### PR TITLE
ucs01 for move and testnet integration

### PR DESCRIPTION
**Pull request by aeryz**

### CODE REVIEW

File: lib/move-cli/Cargo.toml
Comment (benluelo) [inline lib/move-cli/Cargo.toml:1]: whats this crate for? also, since it's a binary, it needs to be under tools/ not lib/

File: move/ethabi
Comment (benluelo) [inline move/ethabi:1]: remove?

File: move/first_tx.json
Comment (benluelo) [inline move/first_tx.json:1]: remove

File: move/second_tx.json
Comment (benluelo) [inline move/second_tx.json:1]: remove

File: voyager/devnet-config.json
Comment (benluelo) [inline voyager/devnet-config.json:1]: revert non-devnet changes to this file

File: voyager/modules/chain/movement/src/main.rs [hunk 1]
```diff
@@ -1146,60 +1150,68 @@ impl ChainModuleServer<ModuleData, ModuleCall, ModuleCallback> for ModuleServer<
...
-            key = hex::encode(bcs::to_bytes(&path.to_string().as_bytes()).expect("won't fail")),
-            address = address,
-            height = at.revision_height,
-        );
-
-        let (state_value, proof): (
-            Option<aptos_types::state_store::state_value::StateValue>,
-            aptos_types::proof::SparseMerkleProof,
-        ) = client.get(req).send().await.unwrap().json().await.unwrap();
+        // let req = format!(
```
Comment (benluelo) [inline voyager/modules/chain/movement/src/main.rs:1153]: remove comment?

File: voyager/modules/consensus/movement/src/main.rs [hunk 2]
```diff
@@ -174,18 +174,51 @@ impl QueueInteractionsServer<ModuleData, ModuleCall, ModuleCallback> for ModuleS
...
-                        self.ctx.movement_rest_url, to
-                    ))
-                    .send()
-                    .await
-                    .unwrap()
-                    .json()
-                    .await
-                    .unwrap();
-
+                // let state_proof: StateProofResponse = client
```
Comment (benluelo) [inline voyager/modules/consensus/movement/src/main.rs:177]: remove comment?

File: lib/move-cli/Cargo.toml
Comment (aeryz) [reply lib/move-cli/Cargo.toml:1]: it's just a failure and a bunch of pain

File: voyager/modules/chain/movement/src/main.rs [hunk 3]
```diff
@@ -400,63 +400,22 @@ impl ChainModuleServer for Module {
...
-            key = hex::encode(bcs::to_bytes(&path.to_string().as_bytes()).expect("won't fail")),
-            address = address,
-            height = at.revision_height,
-        );
-
-        let (state_value, proof): (
-            Option<aptos_types::state_store::state_value::StateValue>,
-            aptos_types::proof::SparseMerkleProof,
-        ) = client.get(req).send().await.unwrap().json().await.unwrap();
+        // NOTE(aeryz): This only works with Union's custom Movement node. When the following PR is merged,
```
Comment (benluelo) [inline voyager/modules/chain/movement/src/main.rs:405]: link PR

File: voyager/plugins/client-update/movement/src/main.rs [hunk 4]
```diff
@@ -212,22 +212,11 @@ impl PluginServer<ModuleData, ModuleCall, ModuleCallback> for Module {
...
-                        self.movement_rest_url, to
-                    ))
-                    .send()
-                    .await
-                    .unwrap()
-                    .json()
-                    .await
-                    .unwrap();
-
+                // NOTE(aeryz): This only works with Union's custom Movement node. When the following PR is merged,
```
Comment (benluelo) [inline voyager/plugins/client-update/movement/src/main.rs:217]: link PR

**Pull request merged**